.github/CODEOWNERS

```diff
@@ -63,10 +63,9 @@ extension/awsproxy/                                  @open-telemetry/collector-c
 extension/basicauthextension/                        @open-telemetry/collector-contrib-approvers @jpkrohling @svrakitin
 extension/bearertokenauthextension/                  @open-telemetry/collector-contrib-approvers @jpkrohling @pavankrish123
 extension/healthcheckextension/                      @open-telemetry/collector-contrib-approvers @jpkrohling
-extension/httpforwarder/                             @open-telemetry/collector-contrib-approvers @asuresh4
 extension/jaegerremotesampling/                      @open-telemetry/collector-contrib-approvers @jpkrohling
 extension/oauth2clientauthextension/                 @open-telemetry/collector-contrib-approvers @pavankrish123 @jpkrohling
-extension/observer/                                  @open-telemetry/collector-contrib-approvers @asuresh4 @jrcamp
+extension/observer/                                  @open-telemetry/collector-contrib-approvers @jrcamp
```

Comment:
@jrcamp is also not active here anymore. Feel free to put @dmitryax and @rmfitzpatrick here